contrib/epee/src/readline_buffer.cpp

```diff
@@ -16,6 +17,8 @@ static std::string last_prompt;
 std::mutex line_mutex, sync_mutex, process_mutex;
 std::condition_variable have_line;
 
+std::vector<std::string> rdln::readline_buffer::completion_commands = {"exit"};
```

Comment:
> The variables are private but accessed via public static functions that if invoked during the namespace initialization process will result in undefined behavior. 

There is only one public static function and that is a _class_ level public static function accessing a _class_ level private static variable. As they are in the same translation unit the variable is guaranteed to be defined.

Static initialization order fiasco simply does not apply here.